src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const titleCase = (text: string): string => {
+  return text
+    .replace("_", " ")
```

Comment:
I think this might be an issue with e.g. `hot_air_balloon` since I think `replace` only replaces the first instance of it?

It looks like we can use `replaceAll` here - I know the webapp wouldn't allow it 😭

Reply:
Done 😄